src/test/java/signupTests.java

```diff
@@ -0,0 +1,47 @@
+package com.CSC207.connecthub;
```

Comment:
I noticed that Karth organized his tests by creating a use_case package under test/java and then a specific getpost package within use_case to group his tests. It might be a good idea to follow a similar structure, so all the tests are organized consistently in the same package.